src/legacy/core_plugins/telemetry/server/telemetry_config/get_telemetry_notify_user_about_optin_default.test.ts

```diff
@@ -40,37 +40,50 @@ describe('getNotifyUserAboutOptInDefault: get a flag that describes if the user
         configTelemetryOptIn: false,
       })
     ).toBe(false);
+  });
 
+  it('should return false if user has seen notice', () => {
     expect(
       getNotifyUserAboutOptInDefault({
-        allowChangingOptInStatus: false,
-        telemetrySavedObject: null,
-        telemetryOptedIn: true,
+        allowChangingOptInStatus: true,
+        telemetrySavedObject: { userHasSeenNotice: true },
+        telemetryOptedIn: false,
         configTelemetryOptIn: true,
       })
     ).toBe(false);
-  });
 
-  it('should return false if user has seen notice', () => {
     expect(
       getNotifyUserAboutOptInDefault({
         allowChangingOptInStatus: true,
         telemetrySavedObject: { userHasSeenNotice: true },
-        telemetryOptedIn: false,
-        configTelemetryOptIn: false,
+        telemetryOptedIn: true,
+        configTelemetryOptIn: true,
       })
     ).toBe(false);
+  });
 
+  it('not show notice for users already opted in and has not seen notice yet', () => {
     expect(
       getNotifyUserAboutOptInDefault({
         allowChangingOptInStatus: true,
-        telemetrySavedObject: { userHasSeenNotice: true },
+        telemetrySavedObject: { userHasSeenNotice: false },
         telemetryOptedIn: true,
         configTelemetryOptIn: true,
       })
     ).toBe(false);
   });
 
+  it('should see notice if they are merely opted in by default and have not yet seen the notice', () => {
+    expect(
+      getNotifyUserAboutOptInDefault({
+        allowChangingOptInStatus: true,
+        telemetrySavedObject: { userHasSeenNotice: false },
```

Comment:
@joelgriffith @Bamieh are we using the wrong flag for this?

We have `lastVersionChecked`, which is a string, but would that be the more appropriate info to reference here?